packages/gatsby-plugin-gatsby-cloud/src/utils/trackEvent.js

```diff
@@ -0,0 +1,36 @@
+const pkgjson = require(`../package.json`)
+
+export default async function trackEvent({
+  eventType,
+  orgId,
+  siteId,
+  buildId,
+  name,
+}) {
+  if (process.env.GATSBY_ANALYTICS_ENDPOINT) {
+    try {
+      const body = {
+        time: new Date(),
+        eventType,
+        componentId: `gatsby-plugin-gatsby-cloud_preview-indicator`,
+        version: 1,
+        componentVersion: pkgjson.version,
+        orgId,
```

Comment:
this needs to be organizationId, sorry I might have cheated in the docs.
```suggestion
        organizationId: orgId,
```